wadsrc/static/language.csv

```diff
@@ -3480,7 +3480,6 @@ Textures enabled,GLTEXMNU_TEXENABLED,,,,Povolit textury,Texturen an,,Teksturoj A
 Texture Filter mode,GLTEXMNU_TEXFILTER,,,,Režim filtrování textur,Texturfiltermodus,,Reĝimo por Teksturfiltrado,Modo de filtro de texturas,,Pintakuviointien suodatustapa,Mode de Filtrage Texture,,Modalità filtro texture,テクスチャーフィルター モード,텍스쳐 필터 모드,Textuur Filter mode,Tryb Filtrowania Tekstur,Modo de filtragem de textura,,Mod filtrare texturi,Фильтрация текстур,Текстурни филтер мод
 Anisotropic filter,GLTEXMNU_ANISOTROPIC,,,,Anisotropické filtrování,Anisotropische Filterung,,Anizotropa Filtro,Filtro anisotrópico,,Anisotrooppinen suodatus,Filtre Anisotropique,,Filtro anisotropico,異方性フィルター,이방성 필터,Anisotroop filter,Filtr anizotropowy,Filtragem anisotrópica,,Filtrare anizotropă,Анизотропная фильтрация,Анизотропни фолтер
 Texture Format,GLTEXMNU_TEXFORMAT,,,,Formát textur,Texturformat,,Aranĝo de Teksturoj,Formato de textura,,Pintakuvioinnin tiedostomuoto,Format Texture,,Formato texture,テクスチャー フォーマット,텍스쳐 포맷,Textuur Formaat,Format Tekstur,Formato de Textura,,Format Texturi,Формат текстур,Формат текстура
-Enable hires textures,GLTEXMNU_ENABLEHIRES,,,,Povolit textury ve vysokém rozlišení,Hochauflösende Texturen an,,Aktivigi Altdistingivajn Teksturojn,Activar texturas de alta resolución,,Salli korkean erotuskyvyn pintakuvioinnit,Activer Textures haute résolution,,Abilita texture alta qualità,ハイレゾ テクスチャー有効,고해상도 텍스쳐 사용,Hoge-resolutie texturen op,Włącz tekstury wysokiej jakości,Habilitar texturas de alta resolução,,Texturi de înaltă rezoluție,Текстуры с высоким разрешением,Омогући текстуре велике резолуције
```

Comment:
This change will be overwritten after the next language export. There is no need to do it by hand, just mention that `GLTEXMNU_ENABLEHIRES` is no longer used.